ONSdigital/eq-author-app
# EAR-1772-repeating-individual-answers

### What is the context of this PR?

This PR is linked to this [Jira ticket](https://jira.ons.gov.uk/browse/EAR-1772)


>Added repeating individual answers toggle


### How to review

> Add `repeatingIndividualAnswers` to the feature flags

- Create a questionnaire with a list collector page, then add another question page
  - add a basic answer and ensure that the repeating individual answers toggle is present, it should be the last item on the page
  - then, turn on the toggle and ensure that the `Add another answer` button is disabled and the toggle status is saved 
- Check that there is validation against the linked collection list as well as a piped answer in the answer label
 - the answer label should have two validation messages, refer to the prototype
- Ensure that when a collection list is selected the validation error goes away
- Navigate to the answer label and click the pipe answer button
  -  ensure that you can pipe an answer from a linked collection list and that this action resolves the second validation 
 - check that the preview page matches the figma prototype
-Ensure that when there are multiple basic answer types on a single page the repeating label and input toggle is disabled
-Check when importing a questionnaire there are no errors
